mitodl/mitxonline
# Added course index page

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #79

#### What's this PR do?
Adds a course index page, adds a data migration that moves existing course pages under that index page, and changes the URL rules such that course pages are now accessed at `/courses/<readable_id>/`

#### How should this be manually tested?
- Run migrations, and make sure all of your course pages are children of the course index page, which is a child of the home page
- Add a new course page under the course index page in Wagtail to make sure that works
- Confirm that the URLs for course pages are as described

#### Screenshots (if appropriate)
<img width="656" alt="ss 2021-08-10 at 20 40 46 " src="https://user-images.githubusercontent.com/14932219/128957293-9f3786eb-addc-4e83-86ba-91ffb0ba7f51.png">

<img width="621" alt="ss 2021-08-10 at 22 45 31 " src="https://user-images.githubusercontent.com/14932219/128966934-874e9c41-33d9-4531-a668-e8f148fe39d8.png">
